classes/services/PKPAuthorService.inc.php

```diff
@@ -228,9 +228,8 @@ public function validate($action, $props, $allowedLocales, $primaryLocale)
      */
     public function add($author, $request)
     {
-        $authorDao = DAORegistry::getDAO('AuthorDAO'); /** @var AuthorDAO $authorDao */
-        $authorId = $authorDao->insertObject($author);
-        $author = $this->get($authorId);
+        $authorId = $this->dao->insert($author);
+        $author = Repo::author()->get($authorId);
```

Comment:
The `Repository` is meant to replace the `Service` class, so you should be able to remove `PKPAuthorService` and refactor all places where it is used.

Reply:
`PKPAuthorService` and `PKPAuthorQueryBuilder` removed